oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/view/CatsLoadBalancerProvider.groovy

```diff
@@ -49,7 +49,6 @@ class CatsLoadBalancerProvider implements LoadBalancerProvider<AmazonLoadBalance
     this.awsProvider = awsProvider
   }
 
-  @Override
   Map<String, Set<AmazonLoadBalancer>> getLoadBalancers() {
```

Comment:
and here - if you'd prefer each provider do the actual cleanup that is fine, but I'm ok with you killing off the impl in this class